.readthedocs.yaml

```diff
@@ -10,7 +10,6 @@ build:
 python:
     install:
         - requirements: docs/requirements.txt
-    system_packages: true
```

Comment:
This option became deprecated on RTD and prevented the documentation build from completing